cmd/tailscale/cli/ping.go

```diff
@@ -50,6 +50,7 @@ relay node.
 		fs.BoolVar(&pingArgs.verbose, "verbose", false, "verbose output")
 		fs.BoolVar(&pingArgs.untilDirect, "until-direct", true, "stop once a direct path is established")
 		fs.BoolVar(&pingArgs.tsmp, "tsmp", false, "do a TSMP-level ping (through IP + wireguard, but not involving host OS stack)")
+		fs.BoolVar(&pingArgs.icmp, "icmp", false, "do a ICMP-level ping (through IP + wireguard. Remote host OS stack only)")
```

Comment:
I don't like having two args here, but I also wasn't too keen on breaking the UI interface - what do you think?

Reply:
it's fine for now. Later we'll make a mode that by default with no flags does everything and prints a nice report.